.github/workflows/release.yml

```diff
@@ -99,6 +99,7 @@ jobs:
       - build_linux
     name: Create Release
     steps:
+      - uses: actions/checkout@v2
```

Comment:
do we need the `name: Checkout` line like the other checkout steps?

Reply:
Not really, but it makes it "look nicer". I can add it.